### ISSUE TITLE
Select multiple chats

### ISSUE THREAD

**Issue by gabrielfern** [telegramdesktop/tdesktop#4079]
Hey guys, this is a enhancement to the user experience. Today to leave a group/delete some chat or whatever in that way, you need to do that manually every chat you have, it would be a great if in some how we could select multiple chats to apply some action to all of them. Or am i unaware of how to do that? Imagine if you had a lot of dead chats, how to get away of all of them in a don't bother way?

**Comment by tdesktop-closer** [comment]
Hey there!

We're automatically closing this issue since there was no activity in this issue since 379 days ago. We therefore assume that the user has lost interest or resolved the problem on their own. Closed issues that remain inactive for a long period may get automatically locked.

Don't worry though; if this is in error, let us know with a comment and we'll be happy to reopen the issue.

Thanks!

<sub>(Please note that this is an [automated](https://github.com/auchri/github_auto_closer) comment.)</sub>

**Comment by Aokromes** [comment]
still wanted

**Comment by Aokromes** [comment]
still wanted

**Comment by iamkarlson** [comment]
any updates?

**Comment by theLockesmith** [comment]
This would be a great feature. It already exists on mobile, why not desktop clients as well?

**Comment by brenio55** [comment]
Still interesting to add!

**Comment by rabyj** [comment]
yeah that's a pretty basic feature to be able to copy paste to another medium

**Comment by yuganoffl** [comment]
is this issue resolved?

**Comment by brenio55** [comment]
Yess!!! They solved it, thank you guys!

**Comment by rabyj** [comment]
@gabrielfern Can you confirm the enhancement is present and then close the issue please?
